projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -359,13 +359,14 @@ public function render_button( $attributes, $content = null, $block = null ) {
 			return $this->render_button_preview( $attributes, $content );
 		}
 
-		if ( empty( $attributes['planId'] ) ) {
+		if ( empty( $attributes['planId'] ) && empty( $attributes['planIds'] ) ) {
 			return $this->render_button_error( new WP_Error( 'jetpack-memberships-rb-npi', __( 'No plan was configured for this button.', 'jetpack' ) . ' ' . __( 'Edit this post and confirm that an existing payment plan is selected for this block.', 'jetpack' ) ) );
 		}
 
 		// This is string of '+` separated plan ids. Loop through them and
 		// filter out the ones that are not valid.
-		$plan_ids    = explode( '+', $attributes['planId'] );
+		// phpcs:ignore Universal.Operators.DisallowShortTernary.Found
+		$plan_ids    = $attributes['planIds'] ?: explode( '+', $attributes['planId'] );
```

Comment:
```suggestion
		$plan_ids    = empty( $attributes['planIds'] ) ? explode( '+', $attributes['planId'] ) : $attributes['planIds'];
```
Or you could use `??` (which doesn't have a PHPCS rule complaining about it) in place of `?:` if we're not worried that planIds might come in as an empty array somehow.

Reply:
I expanded it out.